packages/graphql/package.json

```diff
@@ -43,6 +43,7 @@
     "apollo-link-schema": "^1.2.3",
     "apollo-link-ws": "^1.0.18",
     "apollo-server": "^2.6.7",
+    "bignumber.js": "9.0.0",
```

Comment:
Just checking, did you run "lerna add bignumber.js --scope=@origin/graphql" ?
It should also update your yarn.lock which is a file I think we commit to the repo.
Though bignumber.js may already be a dependency of graphql since it is a web3 depends on it.